cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -146,7 +145,7 @@ func (m *MasqDaemon) Run() {
 	// Periodically resync to reconfigure or heal from any rule decay
 	for {
 		func() {
-			defer time.Sleep(time.Duration(m.config.ResyncInterval))
+			defer time.Sleep(time.Duration(*resyncInterval))
```

Comment:
I think this will be interpreted as 60 _nanoseconds_. That is the base time value for `Duration` https://cs.opensource.google/go/go/+/refs/tags/go1.17.5:src/time/time.go;l=588

```suggestion
			defer time.Sleep(*resyncInterval * time.Second)
```

Reply:
Nice catch! Fixed.